<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget_showRes" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_getFrameInfo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GetFrameInfo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameNumber">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>frame number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_frameNumber">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showRes">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>showResult</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_setWkPath">
     <property name="text">
      <string>SetPath</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dataSetPath">
     <property name="text">
      <string>set dataset path</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_saveImg">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>saveImg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setSavePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>setImgeSavePath</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_savePath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>550</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>images save path</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <addaction name="actionImageCalc"/>
   </widget>
   <addaction name="menuAlgorithm"/>
  </widget>
  <action name="actionImageCalc">
   <property name="text">
    <string>ImageSimulation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
